### ISSUE TITLE
After sat 13.00 lets disable walking/cycling

### ISSUE THREAD

**Issue by AnnaGoodman1** [Active-Travel-Academy/ata-traffic-recorder#16]
The 'infrequently_routed" ones ran correctly on Tues BUT ran x3 in immediate succession, rather than x1, on saturday

Try:

select * from journey_runs where journey_id=5052 ;

shows 3 runs as 13.00 (12.00 GMT) rather than just one

**Comment by nikolai-b** [comment]
Isn't this by design?

```sql
ltns=> select journey_runs.created_at, runs.mode from journey_runs join runs on journey_runs.run_id = runs.id where journey_id=5052 order by created_at;
          created_at           |   mode    
-------------------------------+-----------
 2021-05-18 07:30:04.998491+00 | driving
 2021-05-18 12:00:05.121557+00 | driving
 2021-05-18 16:30:05.371816+00 | driving
 2021-05-22 12:00:05.445304+00 | bicycling
 2021-05-22 12:00:05.447479+00 | walking
 2021-05-22 12:00:05.44929+00  | driving
```

The reason it didn't happen on Tue was we hadn't done https://github.com/Active-Travel-Academy/ata-traffic-recorder/issues/14 so walking and driving weren't working on Tue.

**Comment by AnnaGoodman1** [comment]
ok, but hmm, I had ideally wanted this only the first 7 days of the month to do walking and cycling. Just as there are 5600 journyes that this will affect and it is expensive. so much so that I now think once per month is enough .

Sorry, could we wait until after it has run on Saturday at 13.00 and then hard re-write the code to *disable the x3* before Tuesday ? so that I have it once for walking and cycling, but not loads of times.  and then I won't need walking/cycling again until next June so we can again do an enable/disable bodge?

**Comment by nikolai-b** [comment]
I've spent a while working out how to solve this and got a way.  I've pushed the fix but I won't update when it actually runs on the Tue and Sat only if they are in the first 7 days of the month.  It is quite an obscure fix and I'd like to test it but it isn't easy...

So you want it to run this Sat 29th of March?  Next Tue is the 1st of June so it will run then again, can we not just push the fix now and skip this Sat as it is due to run Tue 1st and Sat 5th of June as intended?
